.github/workflows/cygwin.yml

```diff
@@ -0,0 +1,73 @@
+name: Test on Cygwin
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+jobs:
+  cygwin_build_test:
+    runs-on: windows-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: recursive
+          fetch-depth: 0
+      - name: Install Cygwin
+        uses: egor-tensin/setup-cygwin@v3
+        with:
+          platform: x64
+          install-dir: 'C:\tools\cygwin'
+          packages: python37-devel python37-pytest python37-hypothesis python37-cython python37-pip python37-wheel python37-setuptools liblapack-devel libopenblas gcc-fortran git
+      - name: Set Windows PATH
+        uses: egor-tensin/cleanup-path@v1
+        with:
+          dirs: 'C:\tools\cygwin\bin'
+      - name: Verify that bash is Cygwin bash
+        run: |
+          command bash
+          bash -c "uname -svrmo"
+      - name: Update with Cygwin git
+        # fetch-depth=0 above should make this short.
+        run: |
+          bash -c "type git; git fetch --all -p"
+      - name: Verify python version
+        # Make sure it's the Cygwin one, not a Windows one
+        run: |
+          bash -c "type python3.7; python3.7 --version -V"
+      - name: Build NumPy wheel
+        run: |
+          bash -c "python3.7 setup.py bdist_wheel"
+      - name: Install new NumPy
+        run: |
+          bash -c "python3.7 -m pip install dist/numpy-*cp37*.whl"
+      - name: Check that pip installed the extensions
+        run: |
+          bash -c "python3.7 -m pip show numpy"
+          bash -c "python3.7 -m pip show -f numpy | grep .dll"
+          echo >list_dlls_dos.sh 'site_packages=$(python3.7 -m pip show numpy | grep Location | cut -d " " -f 2 -);
+          dll_list=$(for name in $(python3.7 -m pip show -f numpy | grep -F .dll); do echo ${site_packages}/${name}; done)
+          ls -l ${dll_list}
+          file ${dll_list}
+          ldd ${dll_list}
+          cygcheck ${dll_list}
+          cd dist/
+          for name in ${dll_list};
+          do
+              echo ${name}
+              python3.7 -c "import "$(echo ${name} | sed -E -e "s/\/+(home|usr).*?site-packages\/+//g" -e "s/\//./g" -e "s/.cpython-3.m?-x86(_64)?-cygwin.dll//g")
```

Comment:
Is it possible to break this line (and similar) with `\`?

Reply:
Some places, yes, other places, not easily.

I could try using `runtests.py` again; that should work now and would be shorter than the current `np.test()` line